package.json

```diff
@@ -20,7 +20,9 @@
     "vibe-check": "npx vite-bundle-visualizer",
     "generate-openapi-client": "openapi-generator-cli generate -i osidb-3.4.0.oas3.json --output src/generated-client -g typescript-axios",
     "mock-osidb-server": "prism mock osidb-3.0.0.oas3.json",
-    "lint": "eslint . --ext .js,.ts,.vue"
+    "lint": "yarn lint-style && yarn lint-ts-vue",
+    "lint-ts-vue": "eslint . --ext .js,.ts,.vue",
+    "lint-style": "npx stylelint '**/*.{vue,css,scss}'"
```

Comment:
What is the difference between these two?

Reply:
First one (eslint) is for linting ts/js code and second one (stylelint) is for css/scss code

Reply:
Thanks!